tsk/base/tsk_unicode.c

```diff
@@ -580,18 +580,16 @@ tsk_cleanupUTF8(char *source, const char replacement)
  * @param replacement Character to insert into source as needed.
  */
 void
-tsk_cleanupUTF16(TSK_ENDIAN_ENUM endian, wchar_t *source, const wchar_t replacement) {
+tsk_cleanupUTF16(TSK_ENDIAN_ENUM endian, wchar_t *source, size_t wstrlen, const wchar_t replacement) {
```

Comment:
I'd call this source_len instead of wstrlen. That looks too much like a function name.

Reply:
Resolved in commit https://github.com/sleuthkit/sleuthkit/pull/2631/commits/d96828ac84f28bf78675ad312f338097fc2b039b